ui/widget.js

```diff
@@ -367,16 +369,33 @@ $.Widget.prototype = {
 		return this;
 	},
 	_setOption: function( key, value ) {
+		var classKey, elements;
+		if ( key === "classes" ) {
+			for ( classKey in value ) {
+				if ( value[ classKey ] !== this.options.classes[ classKey ] ) {
+					if ( this.classObject[ classKey ] && this.classObject[ classKey ].length ) {
+						elements = $( this.classObject[ classKey ].get() );
```

Comment:
Regarding the variable, just set `elements = this.classObject[ classKey ]` before the first condition that uses it. The property value needs to change to a new instance whenever it's modified anyway, so holding the reference should be fine.